Dockerfile.builder

```diff
@@ -28,7 +28,7 @@ RUN chmod 777 -R ${HOME}
 RUN apt-get update \
     && apt-get install --no-install-recommends -y -q \
     curl wget git unzip bzip2 jq \
-    libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+    libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
     # ^^ additional Cypress dependencies: https://docs.cypress.io/guides/guides/continuous-integration.html#Dependencies
 
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl && \
```

Comment:
We might update this to 1.18 while we're updating the image.

Reply:
ok, updated.